spec/features/saml/loa3_sso_spec.rb

```diff
@@ -83,8 +116,10 @@ def sign_out_user
         loa3_sp_session
 
         visit verify_path
-        click_on t('links.cancel')
-        click_on t('idv.buttons.cancel')
+        expect(page).not_to have_content(t('links.cancel'))
```

Comment:
I thought we were bringing back the cancel link, no?

Reply:
Hmmm we don't, but I think this test is still passing because the link is `links.cancel_verification`. So this line can probably be removed.